network-diagrams/TDR.md

```diff
@@ -0,0 +1,37 @@
+# TDR Network Diagram
+
+![diagram]
+
+[diagram]: TDR.svg "TDR Network Diagram"
+
+This can be broken down by the function of each section
+
+## MVC App
+There is an ECS cluster containing one service which runs a single Fargate container. The container runs a docker image with the Play app installed. The app uses Cognito for authentication.
+
+The cluster is in a private subnet. Inbound traffic comes from the application load balancer through a network interface and outbound traffic goes out over the nat gateway.
+
+Route 53 is setup to send traffic to the load balancer. 
+
+## Graphql Server
+There is an ECS cluster containing one service which runs a single Fargate container. The container runs a docker image with the Sangria app installed.
+
+Route 53 is setup to send traffic to one of two API Gateway endpoints. One if these uses Cognito for authentication and the other uses IAM. The request is sent through a VPC link to the network load balancer
```

Comment:
Fixed

Reply:
In fact, I've taken it out because at the moment, without Cognito, we don't need the second endpoint. I'll add something back in if we go down the Cognito route.